diffsims/tests/crystallography/test_reciprocal_lattice_vector.py

```diff
@@ -72,12 +74,20 @@ def test_init_raises(self, nickel_phase):
         with pytest.raises(ValueError, match="Exactly one of "):
             _ = ReciprocalLatticeVector(nickel_phase)
 
+    @pytest.mark.parametrize("include_zero_beam", [True, False])
```

Comment:
Renaming the variable to `include_vector_vector` would be in line with the parameter name, and would be preferable (in both tests).